### PR TITLE
app-emulation/vagrant: bump to 1.8.4

### PR DESCRIPTION
**Pull request by hydrapolic**
Thanks to Szymon Szypulski for help.

### CODE REVIEW

**Comment by gktrk** [comment]
@gentoo/proxy-maint

File: app-emulation/vagrant/vagrant-1.8.4.ebuild [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+
+RUBY_FAKEGEM_EXTRADOC="CHANGELOG.md README.md"
+RUBY_FAKEGEM_GEMSPEC="vagrant.gemspec"
+RUBY_FAKEGEM_EXTRAINSTALL="keys plugins templates version.txt"
+RUBY_FAKEGEM_TASK_DOC=""
+
+inherit bash-completion-r1 ruby-fakegem eutils
+
+DESCRIPTION="A tool for building and distributing development environments"
+HOMEPAGE="http://vagrantup.com/"
```
Comment (gktrk) [inline app-emulation/vagrant/vagrant-1.8.4.ebuild]: let's switch to https here

**Comment by hydrapolic** [comment]
Thanks @gktrk for the hint, sorry I needed to do some other stuff meanwhile, thanks for merging.

**Comment by gktrk** [comment]
Thanks @monsieurp for merging it

**Pull request closed without merging**